### PR TITLE
[HOLD FOR PREVIEW]移动视图兼容

### PR DESCRIPTION
**Pull request by crypt0sh0j0**
原理：识别UA
Preview @ https://5bdafeffb13fb12639d622cc--heuristic-wright-542fe0.netlify.com/#/

### CODE REVIEW

File: src/components/SponsorPaymentModal.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
         </div>
       </div>
       <div class="columns">
-        <div class="column content is-hidden-mobile">
+        <div class="column content is-hidden-mobile" v-if="!isPhone">
```
Comment (crazycoder1988) [inline src/components/SponsorPaymentModal.vue]: `is-hidden-mobile` is enough to hide in mobile, why ` v-if="!isPhone"`?

File: src/components/SponsorPaymentModal.vue [hunk 2]
```diff
@@ -48,47 +66,52 @@
 </template>
 
 <script>
-import { mapState, mapActions } from 'vuex';
-import SimpleWallet from '@/libs/SimpleWallet';
-import API from '@/util/api';
-import QrCode from '@xkeshi/vue-qrcode';
+import { mapState, mapActions } from "vuex";
```
Comment (crazycoder1988) [inline src/components/SponsorPaymentModal.vue]: run `npm run lint`.

File: src/components/SponsorPaymentModal.vue [hunk 3]
```diff
@@ -33,10 +33,28 @@
             Pay with Scatter
           </button>
         </div>
-        <div class="column content">
+        <div class="column content is-hidden-mobile" v-if="!isPhone">
           <h4>Pay with Wallet Apps</h4>
-          <p>We support <a href="http://www.mathwallet.org/en/" target="_black">Math Wallet</a>, <a href="https://www.mytokenpocket.vip/en/" target="_black">Token Pocket</a> and <a href="http://meet.one/" target="_black">MEET.ONE</a>. <br>Scan QR code to pay:</p>
-          <QrCode :value="walletTransferData" :options="{ size: 200 }" />
+            <p>We support <a href="http://www.mathwallet.org/en/" target="_black">Math Wallet</a>,
```
Comment (crazycoder1988) [inline src/components/SponsorPaymentModal.vue]: Incorrect indention

File: src/components/SponsorPaymentModal.vue [hunk 4]
```diff
@@ -33,10 +33,28 @@
...
-          <QrCode :value="walletTransferData" :options="{ size: 200 }" />
+            <p>We support <a href="http://www.mathwallet.org/en/" target="_black">Math Wallet</a>, 
+            <a href="https://www.mytokenpocket.vip/en/" target="_black">Token Pocket</a> and 
+            <a href="http://meet.one/" target="_black">MEET.ONE</a>. <br>Scan QR code to pay:</p>
+            <QrCode :value="walletTransferData" :options="{ size: 200 }" />
+        </div>
+
+        <div class="column content" v-if="isPhone">
+          <h4>Pay with Wallet Apps</h4>
+              <button :class="['button', 'is-white', 'is-rounded', 'is-outlined', { 'is-loading': isScatterPaying }]"
```
Comment (crazycoder1988) [inline src/components/SponsorPaymentModal.vue]: Incorrect indention

File: src/components/SponsorPaymentModal.vue [hunk 5]
```diff
@@ -33,10 +33,28 @@
...
           <h4>Pay with Wallet Apps</h4>
-          <p>We support <a href="http://www.mathwallet.org/en/" target="_black">Math Wallet</a>, <a href="https://www.mytokenpocket.vip/en/" target="_black">Token Pocket</a> and <a href="http://meet.one/" target="_black">MEET.ONE</a>. <br>Scan QR code to pay:</p>
-          <QrCode :value="walletTransferData" :options="{ size: 200 }" />
+            <p>We support <a href="http://www.mathwallet.org/en/" target="_black">Math Wallet</a>, 
+            <a href="https://www.mytokenpocket.vip/en/" target="_black">Token Pocket</a> and 
+            <a href="http://meet.one/" target="_black">MEET.ONE</a>. <br>Scan QR code to pay:</p>
+            <QrCode :value="walletTransferData" :options="{ size: 200 }" />
+        </div>
+
+        <div class="column content" v-if="isPhone">
```
Comment (crazycoder1988) [inline src/components/SponsorPaymentModal.vue]: `is-hidden-tablet` should be enough

File: src/components/SponsorPaymentModal.vue (hunk 1 above)
Comment (crypt0sh0j0) [reply src/components/SponsorPaymentModal.vue]: nope in my Windows Chrome

File: src/components/SponsorPaymentModal.vue [hunk 6]
```diff
@@ -48,47 +66,52 @@
...
-    ...mapActions(['loginScatterAsync', 'updateLandInfoAsync']),
+    ...mapActions(["loginScatterAsync", "updateLandInfoAsync"]),
     paidWithWalletApp() {
       this.updateLandInfoAsync();
       this.$dialog.alert({
-        type: 'is-black',
-        title: '恭喜你成功转账',
-        message: '30秒内自动刷新数据，即可确认你是否为新地主。一切以区块链上交易记录为准，购买地皮失败则退款。',
-        confirmText: '好的',
+        type: "is-black",
```
Comment (crazycoder1988) [inline src/components/SponsorPaymentModal.vue]: Prefer to use toast instead

File: src/components/SponsorPaymentModal.vue (hunk 2 above)
Comment (crypt0sh0j0) [reply src/components/SponsorPaymentModal.vue]: okay

File: src/components/SponsorPaymentModal.vue (hunk 1 above)
Comment (crazycoder1988) [reply src/components/SponsorPaymentModal.vue]: Can't catch your point. Your windows Chrome of course won't hide.. You are desktop 🤣

Comment (crazycoder1988) [reply src/components/SponsorPaymentModal.vue]: image

You need to click this to simulate a mobile device (UA + Correct responsive screen size)

File: src/components/SponsorPaymentModal.vue (hunk 6 above)
Comment (crypt0sh0j0) [reply src/components/SponsorPaymentModal.vue]: I don't think that my code 🙄 
👇👇👇👇👇👇
[history](https://github.com/crypto-meetup-dev/cryptomeetup-portal/commit/43d7e75e867973e01d2508110151bbf20e356d66#r31134939)

You started this `SponsorPaymentModal.vue` file with this `alert`, buddy. Can you do this yourself? Thank you

File: src/components/SponsorPaymentModal.vue (hunk 5 above)
Comment (crypt0sh0j0) [reply src/components/SponsorPaymentModal.vue]: okay

File: src/components/SponsorPaymentModal.vue (hunk 6 above)
Comment (crazycoder1988) [reply src/components/SponsorPaymentModal.vue]: It's not my code as well, I just do renames :(

**Pull request merged**